Formula/mbedtls.rb

```diff
@@ -37,6 +37,7 @@ def install
                     "-DUSE_SHARED_MBEDTLS_LIBRARY=On",
                     "-DPython3_EXECUTABLE=#{which("python3")}",
                     "-DCMAKE_INSTALL_RPATH=#{rpath}",
+                    "-DENABLE_TESTING=OFF",
```

Comment:
We shouldn't do this. If the upstream test suite fails that should be reported and not disabled.

Reply:
Yeah, sounds good, let me report this onto the upstream.

Reply:
I've added `ENV.deparallelize` to try out a non-parallel build, as suggested by upstream.

Reply:
Does passing `-DGEN_FILES=OFF` not work?

Reply:
> Pass -DGEN_FILES=OFF to cmake. This should use the generated files from the archive. (And I think it should be the default in the release, both with make and with cmake.)

Based on the comment, I assumed these files would be present in the "final" release archive (https://github.com/ARMmbed/mbedtls/archive/v3.1.0.tar.gz), which isn't identical to the archive we're using (https://github.com/ARMmbed/mbedtls/archive/mbedtls-3.1.0.tar.gz). I'll try this solution locally and update the PR if it works.

Reply:
Ah, the checksum difference is only due to the filenames being different. Using `-DGEN_FILES=OFF` seems to work, so I've updated the PR.